jedi/api/environment.py

```diff
@@ -17,7 +17,7 @@
 
 _VersionInfo = namedtuple('VersionInfo', 'major minor micro')  # type: ignore[name-match]
 
-_SUPPORTED_PYTHONS = ['3.11', '3.10', '3.9', '3.8', '3.7', '3.6']
+_SUPPORTED_PYTHONS = ['3.12', '3.11', '3.10', '3.9', '3.8', '3.7', '3.6']
```

Comment:
Just FYI I'm still thinking about allow users to use newer versions than 3.12, even if they are not officially supported yet. I feel like it's a bit strange, that CPython devs are not able to use a 3.13 Jedi at the moment, even though that is mostly the same as 3.12. But that's probably for another pull request.

Reply:
Agreed.

For Pillow, we let people install for whatever version and platform they want.

But because of the C extensions, you're best of using a prebuilt wheel for Windows, as it's likely to fail for most people who don't have a compiler set up.

So for Windows + 3.13+ we print a warning instead: https://github.com/python-pillow/Pillow/blob/c556c5f2412658a80fceb8f8533d49ba70b319e5/setup.py#L42-L53

And there will be an alpha release next month for 3.13 ([PEP 719](https://peps.python.org/pep-0719/)), it's good to let people test early, it can help find bugs in CPython nice and early.